### PR TITLE
[EH][DDCE-2340] Fixed A and AA accessibility issues

### PR DESCRIPTION
**Pull request by LetMeTestThat**
# DDCE-2340

Accessibility issues identified can be found here: https://github.com/hmrc/accessibility-audits-live/issues/388

Issues addressed:

- Error summary links (A)
- No space between attributes (A)
- No timeout dialogue (A): **Added additional service page to callback when user timeout**
- Text is clipped (AA)

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -25,15 +25,13 @@ object AppDependencies {
...
   )
 
   val test: Seq[ModuleID] = Seq(
-    "org.pegdown" % "pegdown" % "1.6.0",
+    "uk.gov.hmrc" %% "bootstrap-test-play-28" % "5.14.0",
     "com.typesafe.play" %% "play-test" % PlayVersion.current,
-    "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0",
-    "org.scalatestplus" %% "mockito-3-4" % "3.2.9.0",
-    "com.vladsch.flexmark"    %  "flexmark-all"        % "0.36.8"
+    "org.scalatestplus" %% "mockito-3-4" % "3.2.9.0"
```
Comment (Ufuoma-Daniel) [inline project/AppDependencies.scala:34]: mockito 3.2.10.0 is available.

File: conf/application.conf [hunk 2]
```diff
@@ -162,6 +162,7 @@ play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline'
 gg-urls {
   login-callback.url = "http://localhost:8884/lifetime-isa/company-structure"
   logout-callback.url = "http://localhost:8884/lifetime-isa/signed-out"
+  timeout-callback.url = "http://localhost:8884/lifetime-isa/timed-out"
```
Comment (Ufuoma-Daniel) [inline conf/application.conf:165]: Has this new config been overwritten in base or the app-config-env*?

File: project/AppDependencies.scala (hunk 1 above)
Comment (LetMeTestThat) [reply project/AppDependencies.scala:34]: I tried but this doesn't seem to play nicely with flexmark.

File: conf/application.conf (hunk 2 above)
Comment (LetMeTestThat) [reply conf/application.conf:165]: Thanks for the reminder! This is a new key-value so will need to add this to each env config.

Comment (LetMeTestThat) [reply conf/application.conf:165]: https://github.com/hmrc/app-config-base/pull/6199

Comment (Ufuoma-Daniel) [reply conf/application.conf:165]: Just doing base and prod might be easier.

**Comment by Ufuoma-Daniel** [comment]
Don't forget to squash the commits.

File: conf/application.conf (hunk 2 above)
Comment (LetMeTestThat) [reply conf/application.conf:165]: https://github.com/hmrc/app-config-production/pull/12549

**Pull request merged**